doomgrader.sh

```diff
@@ -2,15 +2,40 @@
 
 set -e
 
-# doomgrader config
+# doomgrader config; EDIT HERE!
```

Comment:
I try to avoid all caps and exclamation marks in code because it comes across as shouting

Reply:
I wanted to mark it with something so that users can know where to edit the variables, but I can make it lower case

Reply:
They should know because this is the only place the config variables are set

Reply:
I honestly had trouble when I first ran script, thinking that I needed to change the doomgrader config, internal paths, and steam credential prompt, all within the script. Would you be okay with it just saying?:
`# doomgrader config, edit here`

Reply:
Oh, actually, what about?:
`# doomgrader config, edit DOOMGRADER_ROOT and STEAM_PATH`
I think this may make the script clearer